crash_similarity.py

```diff
@@ -83,14 +83,10 @@ def delete_old_models(current_date, force_train):
     delete all models in case of user forces new training,
     if not, delete all models except of today's model
     """
-    models = [f for f in os.listdir('./models')]
     if force_train:
-        for model in models:
-            os.remove('./models/' + model)
+        [os.remove('./models/' + model) for model in os.listdir('./models')]
     else:
-        for model in models:
-            if current_date not in model:
-                os.remove('./models/' + model)
+        [os.remove('./models/' + model) for model in os.listdir('./models') if current_date not in model]
```

Comment:
And what if we will want to change deletion logic? 
The idea is to have one line with `os.remove` and different logic of list creation. 
```models = [f for f in os.listdir('./models')]
if not force_train:
    models = [m for m in models if current_date not in m]
for m in models:
    os.remove('./models/{}'.format(m))
```